### ISSUE TITLE
check for ggplot2 V3.0.0 compatibility

### ISSUE THREAD

**Issue by wibeasley** [OuhscBbmc/DeSheaToothakerIntroStats#36]
A major version of ggplot2 was released a few weeks ago.  
* https://www.tidyverse.org/articles/2018/07/ggplot2-3-0-0/
* https://github.com/tidyverse/ggplot2/releases/tag/v3.0.0

@deshea, I'm going to see if any code breaks and things look right.  (But it might be a few weeks from now until I can.)  When that's done, do you mind glancing through the graphs yourself to see if anything changed so it's no longer consistent with the text/message?

**Comment by deshea** [comment]
Of course, thanks. No rush. Sales of the book have sucked.

On Aug 13, 2018, at 8:27 AM, Will Beasley <notifications@github.com<mailto:notifications@github.com>> wrote:


A major version of ggplot2 was released a few weeks ago.

  *   https://www.tidyverse.org/articles/2018/07/ggplot2-3-0-0/<https://urldefense.proofpoint.com/v2/url?u=https-3A__www.tidyverse.org_articles_2018_07_ggplot2-2D3-2D0-2D0_&d=DwMCaQ&c=VjzId-SM5S6aVB_cCGQ0d3uo9UfKByQ3sI6Audoy6dY&r=GqhtunBVkx6OpAXhszze4dQ4ITw-G_eu8MLeRF3XsVk&m=eSffRZ7F4NxQfasCz4DvlnnHAxU40ircyBL3kpqRa3Y&s=nXq7HTSrLmJsdJ17kFiEs1U3T9ZIwqGM7BOGqM8IZ5g&e=>
  *   https://github.com/tidyverse/ggplot2/releases/tag/v3.0.0<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_tidyverse_ggplot2_releases_tag_v3.0.0&d=DwMCaQ&c=VjzId-SM5S6aVB_cCGQ0d3uo9UfKByQ3sI6Audoy6dY&r=GqhtunBVkx6OpAXhszze4dQ4ITw-G_eu8MLeRF3XsVk&m=eSffRZ7F4NxQfasCz4DvlnnHAxU40ircyBL3kpqRa3Y&s=LhflY3fskPMriLQLhmUcBntFW5TGgPlMU8QHCjEL0Dk&e=>

@deshea<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_deshea&d=DwMCaQ&c=VjzId-SM5S6aVB_cCGQ0d3uo9UfKByQ3sI6Audoy6dY&r=GqhtunBVkx6OpAXhszze4dQ4ITw-G_eu8MLeRF3XsVk&m=eSffRZ7F4NxQfasCz4DvlnnHAxU40ircyBL3kpqRa3Y&s=GBWDA4HWskrs1jmoQMopG9COFSrqjemkRqFdk_jX6Uc&e=>, I'm going to see if any code breaks and things look right. (But it might be a few weeks from now until I can.) When that's done, do you mind glancing through the graphs yourself to see if anything changed so it's no longer consistent with the text/message?

**Comment by wibeasley** [comment]
@deshea, I ran the code again, and surprisingly nothing broke.  Check out the side-by-side differences with 

https://github.com/OuhscBbmc/DeSheaToothakerIntroStats/commit/151dded705ccb4a203c424d4cf24721ca8aa98b3?short_path=3b52452#diff-3b524529caf6f2fb8a7e21155cdf92c6

Here are the three things I noticed.  Anything you'd like to change?
* the jitters are a little different in 3-21 through 3-23.  Not worse -but different.
* the subscripts in 11.01 are a little looser.  It's not my preference, but probably not off enough to modify the code for.  Future MathJax updates might change it in the future.
* The margins are rendered a little differently --and for the better-- in figures 12-10 through 12-12.

I'm flying tomorrow, and I might update parts of the code.  These have been a big help as a reference point for when we try to remember how to do something.
